client/src/Components/Widget0.js

```diff
@@ -1,141 +1,145 @@
-
-import React, {Component} from 'react';
+import React, { Component } from "react";
 import { Container } from "reactstrap";
-import ReactMapGL, { Marker, Popup, NavigationControl,LinearInterpolator, FlyToInterpolator } from 'react-map-gl';
-import {defaultMapStyle, pointLayer} from '../mapstyle.js';
-import {fromJS} from 'immutable';
+import ReactMapGL, {
+  Marker,
+  Popup,
+  NavigationControl,
+  LinearInterpolator,
+  FlyToInterpolator
+} from "react-map-gl";
+import { defaultMapStyle, pointLayer } from "../mapstyle.js";
+import { fromJS } from "immutable";
 // const TOKEN = 'pk.eyJ1IjoibWlzbGFtNSIsImEiOiJjanVpdG5vZWoxZThsNGZwamJ4Nmxya2o0In0.19pBli659L76GrJaX0JWoA'; //rayhan's token
-const TOKEN = 'pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqdzg5aWxkYzF1azI0OW5uaWVmazhleXUifQ.XAm1dRGmXuRAMSQm0TJKXg'; //ryan baumann's token
+const TOKEN =
+  "pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqdzg5aWxkYzF1azI0OW5uaWVmazhleXUifQ.XAm1dRGmXuRAMSQm0TJKXg"; //ryan baumann's token
```

Comment:
I created a token specifically for this project so we can track usage and users easily - let's use this `pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqd281d2M3bzF4MHczeXA5dm9xdXg4bjAifQ._vwACgp12mC4RBpuSYl1CA`